### DISCUSSION TITLE
Можно ли запустить временную таблицу из дерева?

### DISCUSSION THREAD

**Discussion by ToeraHL** [totumonline/totum-ru-issues-and-discussions#286] [Как это сделать?] [answered]
Здравствуйте!  Возможно ли запустить временную таблицу из дерева? Предавать в нее значения не нужно, только запуск.
Таблицы же не обрабатывают такое событие, как открытие? Так бы можно было бы по нажатию какой-то таблицы или по ярлыку сразу запускать временную

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Кончно. Просто дайте для нее доступ нужной роли и если ее специально не скрывать из дерева то она там будет показываться. Вот например в liv-demo https://n-a7c70ee-78365.demoru.totum.online/Table/22/61

**Comment by ToeraHL** [comment]
Получается, даже если временная таблица не запущена в модальное окно, то ее суть не меняется и она отображается для каждого пользователя индивидуально?

**Reply by ttm-alexeypolunin** [reply]
Угу. И можно Коды действия при добавлении использовать. При каждом вызове из дерева это будет создание новой таблицы. Но живут они всего час. Поэтому как промежуточные таблицы их не используют. Если нужен небольшой промежуточный агрегат - то берется Расчетная в дереве. А Временные при использовании в дереве - это 95% пользовательские эндпойнты.

**Comment by ToeraHL** [comment]
Большое спасибо за ответ
